decidim-conferences/app/queries/decidim/conferences/organization_published_conferences.rb

```diff
@@ -14,7 +14,7 @@ def query
           OrganizationConferences.new(@organization),
           VisibleConferences.new(@user),
           PublishedConferences.new
-        ).query.order(start_date: :desc)
+        ).query.order(weight: :asc)
```

Comment:
Could we add a secondary order by date in case they have the same weights (as in after an upgrade)?
```suggestion
        ).query.order(weight: :asc, start_date: :desc)
```

Reply:
Great idea! Added it on 8848bc314c5a78e200f627427862d508274d9e37